test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionDescriptorProviderTests.cs

```diff
@@ -38,6 +38,20 @@ public void GetDescriptors_GetsDescriptorsOnlyForValidActions()
         }
 
         [Fact]
+        public void GetDescriptors_DisplayNameIncludesAssemblyName()
+        {
+            // Arrange
+            var provider = GetProvider(typeof(PersonController).GetTypeInfo());
+
+            // Act
+            var descriptors = provider.GetDescriptors();
+            var descriptor = descriptors.Single(ad => ad.Name == nameof(PersonController.GetPerson));
+
+            // Assert
+            Assert.Equal($"{typeof(PersonController).FullName}.{nameof(PersonController.GetPerson)} (Microsoft.AspNetCore.Mvc.Core.Test)", descriptor.DisplayName);
```

Comment:
Maybe the assembly name too. `var assemblyName = GetType().GetTypeInfo().Assembly.GetName().Name;` (in case this assembly ever gets renamed)